nifi-api/src/main/java/org/apache/nifi/processor/util/StandardValidators.java

```diff
@@ -843,7 +843,7 @@ public ValidationResult validate(final String subject, final String input, final
             final boolean validSyntax = pattern.matcher(lowerCase).matches();
             final ValidationResult.Builder builder = new ValidationResult.Builder();
             if (validSyntax) {
-                final long nanos = FormatUtils.getTimeDuration(lowerCase, TimeUnit.NANOSECONDS);
+                final long nanos = new TimeFormat().getTimeDuration(lowerCase, TimeUnit.NANOSECONDS);
```

Comment:
`TimeFormat`, does not have any instance variables, Couldn't all the methods be static hence the constructor for it should be private and no instantiation should be allowed.

Reply:
Yeah, I intentionally didn't make the method static, because I was thinking there were more methods, some of which might benefit from providing a timezone, etc. But now that the refactoring is complete, the methods are really only about durations. Probably makes sense to use static methods and name it `DurationFormat` rather than `TimeFormat`.